app/decorators/casa_case_decorator.rb

```diff
@@ -25,6 +25,20 @@ def case_contacts_ordered_by_occurred_at
     object.case_contacts.sort_by(&:occurred_at)
   end
 
+  def case_contacts_latest
+    object.case_contacts.max_by(&:occurred_at)
+  end
+
+  def successful_contacts_this_week
+    this_week = Date.today - 7.days..Date.today
```

Comment:
I thought we already had something like this somewhere... I want to have all this logic in one central place. I will take a look around.

Reply:
I would like it if we could split out a helper or something(?) to hold all our date-related logic like this

Existing in user: 
```
  def recent_contacts_made(days_counter = 60)
    case_contacts.where(contact_made: true, occurred_at: days_counter.days.ago..Date.today).size
  end
```